src/_array_api_conf.py

```diff
@@ -1,30 +1,25 @@
-# Configuration file for the Sphinx documentation builder.
-#
-# This file only contains a selection of the most common options. For a full
-# list see the documentation:
-# https://www.sphinx-doc.org/en/master/usage/configuration.html
+"""
+Base config for all individual Sphinx docs in the array API repo.
 
-# -- Path setup --------------------------------------------------------------
+The array-api repo contains an individual Sphinx doc for each spec version, all
+of which exist in ../spec/. This file is star-imported in the conf.py files of
+these docs, allowing us to standardize configuration accross API versions.
+
+Every conf.py file which star-imports this should define
+
+* `release`, the str YYYY.MM release. Use "DRAFT" for the draft.
+* `sys.modules['array_api']`, the stubs module to use for autodoc.
+"""
+import re
 
-# If extensions (or modules to document with autodoc) are in another directory,
-# add these directories to sys.path here. If the directory is relative to the
-# documentation root, use os.path.abspath to make it absolute, like shown here.
-#
-import os
-import sys
 import sphinx_material
-sys.path.insert(0, os.path.abspath('./API_specification'))
 
 # -- Project information -----------------------------------------------------
 
 project = 'Python array API standard'
-copyright = '2020-2022, Consortium for Python Data API Standards'
+copyright = '2020, Consortium for Python Data API Standards'
```

Comment:
Why was this changed?

Reply:
Missed this when going through merge conflicts likely, so reverted this change.